### ISSUE TITLE
apollo client v3

### ISSUE THREAD

**Issue by productdevbook** [nuxt-community/apollo-module#407]
When updated nuxt apollo module ?

**Comment by negezor** [comment]
I don't think it makes sense to change this module until [vue-apollo](https://github.com/vuejs/vue-apollo) 4 is stable and until the release of [@nuxt/module-utils](https://youtu.be/oZtsiw2rBgQ?t=404) (or [@nuxt/kit](https://www.npmjs.com/package/@nuxt/kit), idk) Note that [vue-cli-plugin-apollo](https://github.com/Akryum/vue-cli-plugin-apollo) has not yet been updated.

If you don't like waiting either, like me. I created a small repository with a demo of Apollo 3 with Composition API. The only thing I didn't do was inject the helpers into the context. All you need is to install @vue/apollo-composable with the https://github.com/vuejs/vue-apollo/pull/1241 fix. Steps for packing a fix, clone the repository:
```ts
yarn install
yarn build
cd packages/vue-apollo-composable
yarn pack
```
After upload "vue-apollo-composable-v4.0.0-alpha.14.tgz" to the release on GitHub and install it as an NPM package.

If you need uploads and subscriptions, they are easily added with [apollo-upload-client](https://github.com/jaydenseric/apollo-upload-client) and [graphql-ws](https://github.com/enisdenjo/graphql-ws) (or not being actively maintained [subscriptions-transport-ws](https://github.com/apollographql/subscriptions-transport-ws))

**Comment by honeyamin** [comment]
@negezor 
there is lots of error cause of this module,
can use vue apollo instead of this nuxt?

**Comment by negezor** [comment]
@honeyamin The solution above is already working for me in production. I haven't noticed any other errors yet.

**Comment by productdevbook** [comment]
@negezor https://github.com/vuejs/vue-apollo/pull/1241 fixed

**Comment by dargmuesli** [comment]
See https://github.com/nuxt-community/apollo-module/issues/413#issuecomment-1199892664 for a related announcement. I'd recommend to continue following this thread's discussion over there.
